tests/migration_test.go

```diff
@@ -540,7 +540,6 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		})
 		Context("with setting guest time", func() {
 			It("[test_id:4114]should set an updated time after a migration", func() {
-				tests.SkipMigrationTestIfRunnigOnKindInfra()
```

Comment:
Does https://github.com/kubevirt/kubevirt/pull/3429 fixes this test?
This test will be fixed by the migration PR (https://github.com/kubevirt/kubevirt/pull/3392).

Reply:
I though you skipped only the ones who had to do with fc32 change.
If you originally skipped more, then it will probably not solve it and we need to depend also on the migration fix.

Do you prefer to add it as a dependency or split the PR?

Reply:
The migration PR removes the skip fro the migration tests.
I would add the removal on the "fc32 upgrade" skips as a commit to the PR solving the issue.

Reply:
This one actually fails due to `SkipIpv6TestsFailingAfterVmImageUpgradeToFc32` as well.
All tests here fail because of the GuestAgent and the Migration with IPv6 problems.